src/Loop.php

```diff
@@ -223,6 +223,15 @@ public static function supports($feature) {
     }
 
     /**
+     * Get the underlying loop handle.
+     *
+     * @return null|object|resource The loop handle the event loop operates on. Null if there is none.
+     */
+    public static function getLoopHandle() {
```

Comment:
Would you agree though that it's not the norm in open source PHP?

I think for the sake of a byte or two it makes sense to be consistent with the rest of the eco-system.